Libraries/Tools/Community/ui_documentation/README.md

```diff
@@ -1 +1,62 @@
-# project://ui_documentation
+Project: Documentation Generator
+Description: Generate project documentation, for Git repository, from QuickCall and Procedure libraries. 
+Category: automation

```

Comment:
this should be library, not automation, correct? it currently exists under Libraries in the rep